src/sphinx_contributors/__init__.py

```diff
@@ -72,14 +72,17 @@ class ContributorsDirective(Directive):
     def run(self):
         limit = self.options.get("limit", 10)
         order = self.options.get("order", "DESC") == "DESC"
+        providers = self.options.get("providers", "github")
```

Comment:
Please add the option ``providers`` as well in the ``option_spec`` object (line 64).